book/Loresmyth; Remarkable Inns & Their Drinks.JSON

```diff
@@ -1,642 +0,0 @@
-{
```

Comment:
I overwrote on my local by the next file I'm sending you.
Don't worry about that. I removed my local because it created a merge problem between local branches and replaced with the one you currently have.